src/library/Uno.Material/Styles/Controls/v2/ProgressBar.xaml

```diff
@@ -7,17 +7,38 @@
 
 	<x:Double x:Key="MaterialProgressBarMinWidth">250</x:Double>
 	<x:Double x:Key="MaterialProgressBarHeight">4</x:Double>
-	
+
+	<ResourceDictionary.ThemeDictionaries>
+
+		<ResourceDictionary x:Key="Default">
+
+			<StaticResource x:Key="MaterialProgressBarForeground"
+							ResourceKey="PrimaryBrush" />
+
+			<StaticResource x:Key="MaterialProgressBarBackground"
+							ResourceKey="SurfaceVariantBrush" />
+		</ResourceDictionary>
+
+		<ResourceDictionary x:Key="Light">
+
+			<StaticResource x:Key="MaterialProgressBarForeground"
+							ResourceKey="PrimaryBrush" />
+
+			<StaticResource x:Key="MaterialProgressBarBackground"
+							ResourceKey="SurfaceVariantBrush" />
+		</ResourceDictionary>
+	</ResourceDictionary.ThemeDictionaries>
+
 	<Style x:Key="MaterialProgressBarStyle"
 		   TargetType="muxc:ProgressBar">
 		<Setter Property="Foreground"
-				Value="{ThemeResource PrimaryVariantDarkBrush}" />
+				Value="{ThemeResource MaterialProgressBarForeground}" />
 		<Setter Property="Background"
-				Value="{ThemeResource PrimaryVariantLightBrush}" />
+				Value="{ThemeResource MaterialProgressBarBackground}" />
 		<Setter Property="MinWidth"
-				Value="{StaticResource MaterialProgressBarMinWidth}" />
+				Value="{ThemeResource MaterialProgressBarMinWidth}" />
 		<Setter Property="Height"
-				Value="{StaticResource MaterialProgressBarHeight}" />
+				Value="{ThemeResource MaterialProgressBarHeight}" />
```

Comment:
If we want these two Double resources to be overridable then we need to put them in the Theme Dictionaries as well, otherwise they can stay as `StaticResource`

Reply:
As long as it's a ThemeResource in the style, the value is overridable. Both on UWP and Uno platforms.

Example with height set to 20 and 40 
![image](https://github.com/unoplatform/Uno.Themes/assets/90481654/764ab0f1-baf5-4fa3-8d79-2e8573fc89ef)

Reply:
In both case, I only declared a resource with the same key in the page's resource dictionary.